tests/test_notify_suppliers_of_new_questions_answers.py

```diff
@@ -0,0 +1,408 @@
+import mock
+import pytest
+from freezegun import freeze_time
+
+from datetime import datetime
+from lxml import html
+
+from dmscripts.notify_suppliers_of_new_questions_answers import (
+    main,
+    get_live_briefs_with_new_questions_and_answers_between_two_dates,
+    get_ids_of_suppliers_who_started_applying,
+    get_ids_of_suppliers_who_asked_a_clarification_question,
+    get_ids_of_interested_suppliers_for_briefs,
+    get_supplier_email_addresses_by_supplier_id,
+    invert_a_dictionary_so_supplier_id_is_key_and_brief_id_is_value,
+    create_context_for_supplier,
+    send_supplier_emails,
+    get_html_content,
+    EMAIL_SUBJECT,
+    EMAIL_FROM_ADDRESS,
+    EMAIL_FROM_NAME
+)
+
+ALL_BRIEFS = [
+    # a brief with no questions
+    {"id": 0, "clarificationQuestions": []},
+
+    # a brief with a question outside of the date range
+    {"id": 1, "clarificationQuestions": [{"publishedAt": "2017-03-22T06:00:00.669156Z"}]},
+
+    # a brief with two questions outside of the date range
+    {"id": 2, "clarificationQuestions": [
+        {"publishedAt": "2017-03-21T06:00:00.669156Z"},
+        {"publishedAt": "2017-03-22T06:00:00.669156Z"}
+    ]},
+
+    # a brief with a question inside of the date range
+    {"id": 3, "clarificationQuestions": [
+        {"publishedAt": "2017-03-23T06:00:00.669156Z"}
+    ], 'title': 'Amazing Title', 'frameworkFramework': 'digital-outcomes-and-specialists'},
+
+    # a brief with two questions inside of the date range
+    {"id": 4, "clarificationQuestions": [
+        {"publishedAt": "2017-03-22T18:00:00.669156Z"},
+        {"publishedAt": "2017-03-23T06:00:00.669156Z"}
+    ], 'title': 'Brilliant Title', 'frameworkFramework': 'digital-outcomes-and-specialists'},
+
+    # a brief with two questions, one of them outside the range and one inside the range
+    {"id": 5, "clarificationQuestions": [
+        {"publishedAt": "2017-03-22T06:00:00.669156Z"},
+        {"publishedAt": "2017-03-23T06:00:00.669156Z"}
+    ], 'title': 'Confounded Title', 'frameworkFramework': 'digital-outcomes-and-specialists'},
+
+    # a brief with questions over the weekend
+    {"id": 6, "clarificationQuestions": [
+        {"publishedAt": "2017-03-17T18:00:00.669156Z"},
+        {"publishedAt": "2017-03-18T06:00:00.669156Z"},
+        {"publishedAt": "2017-03-19T06:00:00.669156Z"},  # Sunday
+        {"publishedAt": "2017-03-20T06:00:00.669156Z"},
+    ]}
+]
+
+FILTERED_BRIEFS = [ALL_BRIEFS[3], ALL_BRIEFS[4], ALL_BRIEFS[5]]
+
+
+MODULE_UNDER_TEST = 'dmscripts.notify_suppliers_of_new_questions_answers'
```

Comment:
It's probably fine as it is.  I was just taken aback by the ALL_CAPS.  It does feel like we shouldn't have to hack these things just to keep linters happy, but that's where we're at it seems.